lazy-pages/src/sys.rs

```diff
@@ -91,7 +91,7 @@ pub unsafe fn user_signal_handler(info: ExceptionInfo) -> Result<(), Error> {
         (gear_page, 1usize, wasm_mem_begin + gear_page.offset())
     };
 
-    let accessed_page = PageNumber::from(((mem as usize - wasm_mem_begin) / gear_ps) as u32);
+    let accessed_page = PageNumber(((mem as usize - wasm_mem_begin) / gear_ps) as u32);
```

Comment:
here and in other places for all lazy-pages dependent stuff as above

Reply:
The same - mathematically `mem` cannot be less then `wasm_mem_begin `